Detectors/FIT/base/FITBaseLinkDef.h

```diff
@@ -15,7 +15,7 @@
 #pragma link off all functions;
 
 #pragma link C++ class o2::fit::Digit+;
-#pragma link C++ class o2::fit::Hit+;
-
+#pragma link C++ class o2::fit::Geometry + ;
+//#pragma link C++ class o2::fit::Hit+;
```

Comment:
This file is likely not needed (There is another file under 'base/src/' of the same kind).